amd_openvx_extensions/amd_rpp/source/ResizeCropbatchPD.cpp

```diff
@@ -241,7 +254,9 @@ vx_status ResizeCropbatchPD_Register(vx_context context)
 	}
 	if (status != VX_SUCCESS)
 	{
-	exit:	vxRemoveKernel(kernel);	return VX_FAILURE; 
- 	}
+	exit:
```

Comment:
Codacy found an issue: [Label 'exit' is not used.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)